mmv1/products/workstations/api.yaml

```diff
@@ -172,3 +172,113 @@ objects:
                 A list of messages that carry the error details.
               item_type: Api::Type::KeyValuePairs
               output: true
+  - !ruby/object:Api::Resource
+    name: "Workstation"
```

Comment:
We'll need to add test coverage for the resource! Both an MMv1 test (https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-test/) for documentation, and a handwritten update test given update is supported.

Reply:
Working on that, first trying to round up https://github.com/GoogleCloudPlatform/magic-modules/pull/7017 - as this is a dependency for the Workstation resource

Reply:
Ah, thanks! I will wait for https://github.com/GoogleCloudPlatform/magic-modules/pull/7017 in that case- I didn't see the dependency initially and figured they might be possible to do in parallel.

Reply:
I could use some help in the other PR, just placed a comment!

Would be nice if we can get cloud workstations into the next release